R/findNLDI.R

```diff
@@ -289,53 +297,55 @@ findNLDI <- function(comid = NULL,
   }
 
   # Ensure nav types are valid
-  bad_nav = !nav %in% c("UM", "UT", "DD", "DM")
+  bad_nav <- !nav %in% c("UM", "UT", "DD", "DM")
```

Comment:
HAH -- I was going to nag you about the = and <-

Reply:
I knew that one was coming :P